packages/carbon-graphs/src/js/helpers/constructUtils.js

```diff
@@ -7,36 +7,19 @@ import utils from './utils';
 
 /**
  * Simple utility that checks if the input is an array or objects or simple object.
- * Iterate over the array and calls the loader callback function.
+ * Iterate over the array and calls the task callback function.
  *
  * @private
```

Comment:
We can remove the private keyword since this utility function is used by multiple files

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/29/commits/9a8bc160aff7086049e8067a939c84e9f19c5775)